.github/workflows/gke.yaml

```diff
@@ -97,7 +97,41 @@ jobs:
           cilium install \
             --cluster-name=${{ env.clusterName }} \
             --wait=false \
-            --restart-unmanaged-pods=false \
+            --restart-unmanaged-pods=true \
```

Comment:
I think we can actually just drop all the `--restart-unmanaged-pods=true` flags since that's the default anyway.

Reply:
ah yeah lets do it.